packages/server/lib/cloud/protocol.ts

```diff
@@ -275,9 +279,9 @@ export class ProtocolManager implements ProtocolManagerShape {
 
   async uploadCaptureArtifact ({ uploadUrl, fileSize, filePath }: CaptureArtifact): Promise<{
     success: boolean
-    fileSize: number
+    fileSize: number | bigint
     specAccess?: ReturnType<AppCaptureProtocolInterface['getDbMetadata']>
-  } | void> {
+  } | undefined> {
     if (!this._protocol || !filePath || !this._db) {
```

Comment:
this codepath will probably go away with improved error messaging - should never get to this point from upload_artifacts if there is no capture artifact to upload